crates/core/c8y_api/src/http_proxy.rs

```diff
@@ -621,6 +632,18 @@ mod tests {
         let config_file = create_test_config_file_with_content(config_content)?;
 
         // Mock endpoint for config upload event creation
+        // Fake an expired JWT token by rejecting the first config upload request
```

Comment:
Isn't it better to have a dedicated test for this retry feature?

Reply:
I agree that I have been a bit lazy here ;-) Will see if I find some time for that.

Reply:
Added with https://github.com/thin-edge/thin-edge.io/pull/1586/commits/73d114f708f378f630804588f36919af788f4e97